src/main/resources/config/spark/tpcds/test_wp1_longevity_iterative.yaml

```diff
@@ -0,0 +1,41 @@
+# Description: WP1: Longevity
+---
+version: 1
+id: wp1_longevity
+phases:
+- id: setup
+  sessions:
+  - tasks:
+    - template_id: setup
+- id: setup_data_maintenance
+  sessions:
+  - tasks:
+    - template_id: setup_data_maintenance
+    - template_id: setup_data_maintenance
+    - template_id: setup_data_maintenance
+    - template_id: setup_data_maintenance
+    - template_id: setup_data_maintenance
+    - template_id: setup_data_maintenance
+    - template_id: setup_data_maintenance
+    - template_id: setup_data_maintenance
+    - template_id: setup_data_maintenance
+    - template_id: setup_data_maintenance
+- id: init
+  sessions:
+  - tasks:
+    - template_id: init
+- id: build
+  sessions:
+  - tasks:
+    - template_id: build
+- id: data_maintenance_1
+  sessions:
+  - tasks:
+    - template_id: data_maintenance_delta_dependent
+      custom_task_executor: DependentTaskExecutor
```

Comment:
In the `custom_task_executor` value, we should use a full classpath: That way we will allow extensibility even without re-compiling LST-Bench. Then use reflection to instantiate the class.
In addition, I think the `custom_task_executor` should be part in the task definition in the library, since it will be the most common that a given task is always executed by a certain executor.
On the other hand, a parameter such as the `batch size` that we were discussing about adding in a follow-up PR, should probably be part of this workload definition so we can change it across tasks.

Reply:
I'm not sure I get the second part of your comment. For me, the custom_task_executor determines how you execute a template and thus is specific to the execution of the task. The batch size similarly determines the execution of the task, so shouldn't it be defined the same way? I know it's specific to the dependent task executor, but we have a similar case for the time travel id that is defined the same way.

Reply:
Those two properties seems slightly different.
Once it is defined, I would expect most (or all?) `custom_task_executor` properties for a certain task template (e.g., `data_maintenance_delta_dependent`) will be the same and thus should be in the library. Some reasons that come to mind to define it as part of the task template definition in the library: 1) there is also a strong dependency on the input files, 2) the definition of a task template without executor seems incomplete, and 3) you would have to specify the executor multiple times in the workload definition.
`batch size` is slightly different since it would be common to redefine it on a per-phase basis. Thus, in contrast to `custom_task_executor`, `batch size` should probably be part of the workload definition indeed.
On the other hand, these patterns maybe call in the longer term for the possibility to define a default value in the template, that then can be overridden in the workload definition itself.

Reply:
This file needs to be updated with new way of defining arguments for custom tasks?

Reply:
Removed this file and added an appropriate version for TPC-DS with dependent data maintenance phases.